qp/plotting.py

```diff
@@ -0,0 +1,253 @@
+"""Functions to plot PDFs"""
+
+import numpy as np
+
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+
+
+def init_matplotlib():
+    """Initialize matplotlib parameters"""
+    mpl.rcParams['text.usetex'] = True
+    mpl.rcParams['mathtext.rm'] = 'serif'
+    mpl.rcParams['font.family'] = 'serif'
+    mpl.rcParams['font.serif'] = 'Times New Roman'
+    mpl.rcParams['axes.titlesize'] = 16
+    mpl.rcParams['axes.labelsize'] = 16
+    mpl.rcParams['savefig.dpi'] = 250
+    mpl.rcParams['savefig.format'] = 'pdf'
+    mpl.rcParams['savefig.bbox'] = 'tight'
+
+
+init_matplotlib()
+
+COLORS = {}
+COLORS['truth'] = 'k'
+COLORS['mix_mod'] = 'k'
+COLORS['gridded'] = 'k'
+COLORS['quantiles'] = 'blueviolet'
+COLORS['histogram'] = 'darkorange'
+COLORS['samples'] = 'forestgreen'
+
+STYLES = {}
+STYLES['truth'] = '-'
+STYLES['mix_mod'] = ':'
+STYLES['gridded'] = '--'
+STYLES['quantiles'] = '--'#(0,(5,10))
+STYLES['histogram'] = ':'#(0,(3,6))
+STYLES['samples'] = '-.'#(0,(1,2))
+
```

Comment:
Add a 'flex' color/style?

Reply:
Good idea, I'll add that to the branch with flex.

Reply:
resolving since flex is no longer covered in this PR